sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -177,6 +179,7 @@ public boolean deactivateUserByLogin(DbSession dbSession, String login) {
     mapper.deleteUserMeasureFilterFavourites(dto.getId());
     mapper.deleteUserProperties(dto.getId());
     mapper.deleteUserRoles(dto.getId());
+    mapper.deletePropertiesMatchingLogin(asList(DEFAULT_ISSUE_ASSIGNEE), dto.getLogin());
```

Comment:
why not delete all the properties related to the user?
are they deleted during the purge?
I suppose they are not deleted because a user can be reactivated and we want to keep the properties?